frontend/app/routes.json

```diff
@@ -248,6 +248,14 @@
                       "en": "/:lang/apply/:id/living-independently",
                       "fr": "/:lang/appliquer/:id/living-independently"
                     }
+                  },
+                  {
+                    "id": "$lang+/_public+/apply+/$id+/apply-for-yourself",
+                    "file": "routes/$lang+/_public+/apply+/$id+/apply-for-yourself.tsx",
+                    "paths": {
+                      "en": "/:lang/apply/:id/apply-for-yourself",
+                      "fr": "/:lang/appliquer/:id/apply-for-yourself"
```

Comment:
This should be translated, otherwise we might forget to do it before going to production.

Reply:
> This should be translated, otherwise we might forget to do it before going to production.

We don't have the French translation yet, can we use a placeholder for now? something like `apply-for-yourself-fr`

Reply:
> This should be translated, otherwise we might forget to do it before going to production.

Used Deepl for now. Will need to update it when the French content is available